.eleventy.js

```diff
@@ -30,6 +30,29 @@ const DEV_MODE = process.env.ELEVENTY_RUN_MODE !== "build" // i.e. serve/watch
 module.exports = function(eleventyConfig) {
     eleventyConfig.setUseGitIgnore(false); // Otherwise docs are ignored
 
+    // Set DEV_MODE_POSTS to true if the deploy URL starts with 'https://deploy-preview-'
+    const DEV_MODE_POSTS = process.env.ELEVENTY_RUN_MODE !== "build" || 
```

Comment:
You should use the env var from Netlify: https://docs.netlify.com/configure-builds/environment-variables/#build-metadata

Reply:
Thanks for this @ZJvandeWeg! It's visible on the Netlify preview now. I tested it with `CONTEXT=production npm run build`, and it doesn't generate the page.

Should we keep the test file with the future date to check if it actually skips the file in production? I'll remove it afterward.